package-lock.json

```diff
@@ -5891,37 +5891,6 @@
         "node": ">=10.0.0"
       }
     },
-    "node_modules/lockfile-lint": {
```

Comment:
I think this change needs to be reverted?

Reply:
Okay, brought this back. I wonder why it keeps going away.